### PR TITLE
os/bluestore: cleanup min_alloc_size; some formatting nits

### PR DESCRIPTION
**Pull request by xiexingguo**

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -3981,9 +3981,6 @@ int BlueStore::_check_or_set_bdev_label(
 
 void BlueStore::_set_alloc_sizes(void)
 {
-  min_alloc_size_order = ctz(min_alloc_size);
-  assert(min_alloc_size == 1u << min_alloc_size_order);
```
Comment (ifed01) [inline src/os/bluestore/BlueStore.cc]: may be move after min_alloc_size_order assignment?

Comment (xiexingguo) [inline src/os/bluestore/BlueStore.cc]: ah, right. Thanks!

**Comment by xiexingguo** [comment]
changelog:
- sanity check min_alloc_size_order & min_alloc_size
- move the ut-fixer into another separate PR (https://github.com/ceph/ceph/pull/15828)

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-24_03:32:34-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**